Makefile

```diff
@@ -0,0 +1,81 @@
+#!make
+
+# import global variables
+CANDIG_HOME ?= ../CanDIGv2
+env ?= $(CANDIG_HOME)/.env
```

Comment:
This is not recommended. What if my CanDIGv2 path is not in the same relative path as this repo?

Reply:
It's one of those "set if not already set" variables, so I could add a note at the top of the file to say that you should set the env var if that's not where your value is.